.github/workflows/publish_staging.yml

```
@@ -40,226 +40,234 @@ jobs:
-
-    - name: install dependencies (macos only)
-      if: matrix.platform == 'macos-latest'
-      run: brew install protobuf cmake
-
-    - name: install dependencies (windows only)
-      if: matrix.platform == 'windows-latest'
-      run: choco install strawberryperl protoc cmake curl cygwin gnuwin32-m4 msys2 make mingw
-
-    - name: get version
-      run: echo "PACKAGE_VERSION=$(node -p "require('./ui/package.json').version")" >> $GITHUB_ENV
+      uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)